File: First Seminar/FirstSeminar/src/main/java/org/sopt/www/FirstSeminar/dto/HealthCheckResponse.java
package org.sopt.www.FirstSeminar.dto;

import lombok.Getter;

@Getter
public class HealthCheckResponse {

    private static final String STATUS_OK = "OK";
    private String status;

    private HealthCheckResponse(String status) {
        this.status = status;
    }

    public static HealthCheckResponse ok() {
        return new HealthCheckResponse(STATUS_OK);
    }
}

File: First Seminar/FirstSeminarKotlin/src/main/kotlin/org/sopt/www/FirstSeminarKotlin/dto/HealthCheckResponse.kt
package org.sopt.www.FirstSeminarKotlin.dto

class HealthCheckResponse private constructor(val status: String) {

    companion object {
        private const val STATUS_OK = "OK"

        fun ok(): HealthCheckResponse {
            return HealthCheckResponse(STATUS_OK)
        }
    }
}
